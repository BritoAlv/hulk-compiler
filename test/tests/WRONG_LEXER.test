function fib(n : number, n : number) : nn + 1;