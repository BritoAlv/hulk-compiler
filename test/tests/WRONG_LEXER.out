There was a parsing error :
Near error on line 0, plus:

0 function fib(n : number, n : number) : nn + 1;
0                                           ^
Trying to recovery from error
----------------
Seq: 
insert true
----------------
----------------
Seq: 
insert base
----------------
----------------
Seq: 
insert base
----------------
----------------
Seq: 
insert self
----------------
----------------
Seq: 
insert string
----------------
----------------
Seq: 
insert false
----------------
----------------
Seq: 
insert id
----------------
----------------
Seq: 
insert true
----------------
----------------
Seq: 
insert number
----------------
----------------
Seq: 
insert base
----------------
----------------
Seq: 
insert number
----------------
----------------
Seq: 
insert self
----------------
----------------
Seq: 
insert true
----------------
----------------
Seq: 
insert true
----------------
----------------
Seq: 
insert string
----------------
----------------
Seq: 
insert true
----------------
----------------
Seq: 
insert false
----------------
----------------
Seq: 
insert false
----------------
----------------
Seq: 
insert base
----------------
----------------
Seq: 
insert string
----------------
----------------
Seq: 
insert id
----------------
----------------
Seq: 
insert number
----------------
----------------
Seq: 
insert self
----------------
----------------
Seq: 
insert string
----------------
----------------
Seq: 
insert self
----------------
----------------
Seq: 
insert true
----------------
----------------
Seq: 
insert base
----------------
----------------
Seq: 
insert true
----------------
----------------
Seq: 
insert string
----------------
----------------
Seq: 
insert false
----------------
----------------
Seq: 
insert base
----------------
----------------
Seq: 
insert self
----------------
----------------
Seq: 
insert true
----------------
----------------
Seq: 
insert id
----------------
----------------
Seq: 
insert string
----------------
----------------
Seq: 
insert base
----------------
----------------
Seq: 
insert id
----------------
----------------
Seq: 
insert base
----------------
----------------
Seq: 
insert id
----------------
----------------
Seq: 
insert false
----------------
----------------
Seq: 
insert self
----------------
----------------
Seq: 
insert number
----------------
----------------
Seq: 
insert true
----------------
----------------
Seq: 
insert string
----------------
----------------
Seq: 
insert string
----------------
----------------
Seq: 
insert base
----------------
----------------
Seq: 
insert number
----------------
----------------
Seq: 
insert string
----------------
----------------
Seq: 
insert false
----------------
----------------
Seq: 
insert self
----------------
----------------
Seq: 
insert self
----------------
----------------
Seq: 
insert number
----------------
----------------
Seq: 
insert true
----------------
----------------
Seq: 
insert number
----------------
----------------
Seq: 
insert base
----------------
----------------
Seq: 
insert base
----------------
----------------
Seq: 
insert id
----------------
----------------
Seq: 
insert string
----------------
----------------
Seq: 
insert false
----------------
----------------
Seq: 
insert base
----------------
----------------
Seq: 
insert id
----------------
----------------
Seq: 
insert string
----------------
----------------
Seq: 
insert number
----------------
----------------
Seq: 
insert false
----------------
----------------
Seq: 
insert self
----------------
----------------
Seq: 
insert false
----------------
----------------
Seq: 
insert number
----------------
----------------
Seq: 
insert string
----------------
----------------
Seq: 
insert base
----------------
----------------
Seq: 
insert true
----------------
----------------
Seq: 
insert id
----------------
----------------
Seq: 
insert base
----------------
----------------
Seq: 
insert true
----------------
----------------
Seq: 
insert false
----------------
----------------
Seq: 
insert false
----------------
----------------
Seq: 
insert self
----------------
----------------
Seq: 
insert id
----------------
----------------
Seq: 
insert true
----------------
----------------
Seq: 
insert number
----------------
----------------
Seq: 
insert false
----------------
----------------
Seq: 
insert base
----------------
----------------
Seq: 
insert number
----------------
----------------
Seq: 
insert id
----------------
----------------
Seq: 
insert self
----------------
----------------
Seq: 
insert number
----------------
----------------
Seq: 
insert id
----------------
----------------
Seq: 
insert id
----------------
----------------
Seq: 
insert false
----------------
----------------
Seq: 
insert id
----------------
----------------
Seq: 
insert self
----------------
----------------
Seq: 
insert true
----------------
----------------
Seq: 
insert string
----------------
----------------
Seq: 
insert self
----------------
----------------
Seq: 
insert true
----------------
----------------
Seq: 
insert false
----------------
----------------
Seq: 
insert number
----------------
----------------
Seq: 
insert id
----------------
----------------
Seq: 
insert false
----------------
----------------
Seq: 
insert self
----------------
----------------
Seq: 
insert string
----------------
----------------
Seq: 
insert id
----------------
----------------
Seq: 
insert string
----------------
----------------
Seq: 
insert false
----------------
----------------
Seq: 
insert base
----------------
----------------
Seq: 
insert true
----------------
----------------
Seq: 
insert number
----------------
----------------
Seq: 
insert number
----------------
----------------
Seq: 
insert id
----------------
----------------
Seq: 
insert string
----------------
----------------
Seq: 
insert self
----------------
----------------
Seq: 
insert number
----------------
----------------
Seq: 
insert self
----------------
